v3/docs/01-getting-started/a-overview/index.mdx

```diff
@@ -1,66 +1,61 @@
-   The JSON file allows you to configure the genesis state of the modules that compose the Substrate
-   Node's runtime, such as: Balances, Staking, and Sudo. You can learn more about running a Substrate
-   node in the [Create Your First Substrate Chain](/tutorials/v3/create-your-first-substrate-chain) and
-   [Start a Private Network](/tutorials/v3/private-network) tutorials.
-
-2. **With Substrate FRAME**: You can easily create your own custom runtime using [FRAME](/v3/runtime/frame)
-   (Framework for Runtime Aggregation of Modularized Entities), which is what is used to build the Substrate Node. This
-   affords you a large amount of freedom over your blockchain's logic, and allows you to configure data types, select
-   from a library of modules (called "pallets"), and even add your own custom pallets. The
-   [Substrate Developer Hub node template](https://github.com/substrate-developer-hub/substrate-node-template) is a
-   helpful starting point for projects like this. To learn more, see the tutorials to
-   [Build a dApp](/tutorials/v3/proof-of-existence) and [Add a Pallet](/tutorials/v3/add-a-pallet).
-
-3. **With Substrate Core**: The entire FRAME system can be ignored, and the runtime can be designed and implemented from
-   scratch. This could be done in _any language_ that can target [WebAssembly](https://webassembly.org/). If the runtime
-   can be made to be compatible with the abstract block authoring logic of the Substrate node, then you can simply
-   construct a new genesis block from your Wasm blob and launch your chain with the existing Rust-based Substrate
-   client. If not, then you will need to alter the client's block authoring logic, and potentially even alter the header
-   and block serialization formats. In terms of development effort, this is by far the most difficult way to use
-   Substrate, but also gives you the most freedom to innovate.
-
-Substrate allows developers to make choices between technical freedom and ease of development in every step of their design
-decisions. The diagram below illustrates the nature of this flexibility.
+To create a custom blockchain, let’s understand the Substrate framework and other elements. The basic unit where blockchain runs is a node. Since blockchain is a decentralized peer-to-peer system, this node runs inside a client and provides all basic components the system needs to function, such as p2p networking, storage, logic for block processing and consensus, and ability to interact with external blockchains.                          
+                          
+### Usage
 
+Substrate allows developers to make choices between technical freedom and ease of development in every step of their design process. The diagram below illustrates the nature of this flexibility (starting from bottom-up).
 ![Technical Freedom vs Development Ease](../../../../src/images/docs/getting-started/technical-freedom.png)
 
+#### 1. Substrate Node
+
```

Comment:
Generally, you should avoid using numbers except in procedures with steps that are taken in a specific order (ordered lists) and, in this case, the numbers don't add any value to the headings.